templates/containers/chiselled/index.html

```diff
@@ -95,14 +122,156 @@ <h2>Production-ready, ultra-small Ubuntu containers</h2>
+      <h3 class="p-heading--5 u-hide--large u-no-margin--bottom">.NET</h3>
+    </div>
+    <div class="col-6 col-medium-5">
+      <table aria-label="A table that shows how different Ubuntu .NET images compare in size">
+        <thead>
+          <tr class="u-hide--small u-hide--medium">
+            <th class="u-no-padding--left" width="30%">Language</th>
+            <th class="u-no-padding--left" width="70%">Package</th>
+            <th class="u-align--right u-no-padding--right" width="90px">Size</th>
+          </tr>
+          <tr class="u-hide--large">
+            <td width="100%" class="p-text--small-caps u-no-padding--left">Package</td>
+            <td width="90px" class="u-align--right p-text--small-caps u-no-padding--right">Size</td>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td class="display-on-large u-no-padding--left" rowspan="3">.NET</td>
+            <td class="dimmed-text u-no-padding--left d-flex u-vertically-center">
+              <div class="chart-legend-box u-hide--large" style="background:#CBA7B8;"></div>
+              Ubuntu .NET image
+            </td>
+            <td class="u-align--right u-no-padding--right"><strong>219MB</strong></td>
+          </tr>
+          <tr>
+            <td class="dimmed-text u-no-padding--left d-flex u-vertically-center">
+              <div class="chart-legend-box u-hide--large" style="background:#923A66;"></div>
+              Chiselled Ubuntu .NET image
+            </td>
+            <td class="u-align--right u-no-padding--right"><strong>116MB</strong></td>
+          </tr>
+          <tr>
+            <td class="u-no-padding--left d-flex u-vertically-center">
+              <div class="chart-legend-box u-hide--large" style="background:#000000;"></div>
+              Chiselled Ubuntu for self contained .NET image
+            </td>
+            <td class="u-align--right u-no-padding--right"><strong>5MB</strong></td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+    <div class="col-6 col-medium-5 col-start-medium-2 col-small-6">
+      <div id="chiselled-dotnet-chart"
+        aria-label="A chart that visualizes how different Ubuntu.NET images compare in size">
+      </div>
+      <div class="u-sv3 u-hide--large"></div>
+    </div>
+  </div>
+  <div class="row">
+    <hr class="u-hide--large" />
+    <div class="col-medium-1 col-small-6 u-hide--large">
+      <h3 class="p-heading--5 u-hide--large u-no-margin--bottom">Java</h3>
+    </div>
+    <div class="col-6 col-medium-5">
+      <table aria-label="A table that shows how different Java containers compare in size">
+        <thead>
+          <tr class="u-hide--small u-hide--medium">
+            <th class="u-no-padding" width="30%"><span class="u-hide">Language</span></th>
+            <th class="u-no-padding" width="70%"><span class="u-hide">Package</span></th>
+            <th class="u-no-padding" width="90px"><span class="u-hide">Size</span></th>
```

Comment:
Ideally we shouldn't have a mix of percentages and `px` but I understand it's pretty hard to adjust it to different screen sizes.